Cargo.toml

```diff
@@ -127,6 +130,7 @@ anyhow = "1.0"
 bincode = "1.3"
 dashmap = "5.4"
 log = "0.4"
+opentelemetry = { version = "0.19", git = "https://github.com/tqwewe/opentelemetry-rust", branch = "cow", features = ["metrics"] }
```

Comment:
Unfortunately this PR uses some changes in opentelemetry-rust which are not yet published.

The two PR's are:
https://github.com/open-telemetry/opentelemetry-rust/pull/1009
https://github.com/open-telemetry/opentelemetry-rust/pull/1018

Reply:
I saw PRs got merged, do they plan to release the newer version soon?

Reply:
Looking at their previous releases, it seems they don't push releases very frequently :(

I've just made a discussion on it, hopefully we can get more insight there
<https://github.com/open-telemetry/opentelemetry-rust/discussions/1031>